handsontable/src/helpers/dom/element.js

```diff
@@ -323,6 +323,15 @@ export function removeClass(element, className) {
   });
 }
 
+/**
+ * Clear all the class names from the provided element.
+ *
+ * @param {HTMLElement} element The element to process.
+ */
+export function clearClasses(element) {
```

Comment:
I'm wondering if we really want a helper for single-line code? 🤔 The advantage of helpers is that they hide more logic or polyfill something behind. It seems a bit more than what’s necessary.

Reply:
You're probably right 👍 When I was creating it, I assumed the code might be a little longer, but it turned out to be a one-liner.